contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,214 @@
+
+## It got good eventually, right?
+
+In 9 months, we built 6 products and did more than 100 meetings with potential users.
+
+The range of ideas we tried to solve looks broad, but the thing that connected all of them was that we focused on problems we felt we'd experienced in our previous professional lives.
+
+So, what did we build?
+
+### 1. Sales Territory Management Tool
+
+At one stage in my life, I was the VP Sales at an enterprise software company. On paper, it looked like a glamorous job - I used to fly around the world with the sales team, to go to important meetings with huge banks, in fancy skyscrapers, like the [ICC in Hong Kong](https://en.wikipedia.org/wiki/International_Commerce_Centre):
+
+![International Commerce Center - a big skyscraper in Hong Kong](../images/blog/pivoting/international-commerce-centre.jpg)
+
+I vividly remember the realization that over 90% of your sales pipeline, roughly, will never close. Even if you're really, really good at sales.
+
+The _majority_ of your time is spent getting nowhere. All those hotels, flights, calls, fastidiously wearing a suit in inappropriately warm weather - very few of those things actually result in anything.
+
+Sidenote: this is why [product led growth](../handbook/growth/strategy) is so much better.
+
+As deals get older, after a while, the probability they close drops off a cliff. If you're not getting anywhere with a potential customer after a few weeks or months of trying, it's going to get worse not better. Yet systems that are the core products of [\$17.1Bn revenue companies](https://en.wikipedia.org/wiki/Salesforce) by default come with a manually selected arbitrary number for the percentage probability.
+
+There comes a certain time, when it's rational to stop focussing on a given customer and to move on - a fresh customer is more likely to close than one that has been repeatedly ignoring you.
+
+We pulled pipeline data from Hubspot or Salesforce, then used predictive analytics to work out how this curve looked based on historic data, then applied it to the current pipeline. Once a deal dropped below a certain threshold, we'd recommend you swap out that target company and pull a new one into the pipeline.
+
+We confused a lot of people with this idea, because we ourselves were confused with whom we were targeting.
+
+We got 15 sales leaders to agree to trying this out, sent them a link, then waited...
+
+and waited...
+
+just _one_ person even clicked the sign up link. The rest didn't even try it.
+
+With hindsight, it was way overpowered for tiny teams and we'd only have had a great fit for huge ones with a lot of data. The only people interested in smaller teams were, to use [Crossing the Chasm's framework](https://en.wikipedia.org/wiki/Crossing_the_Chasm), innovators, which doesn't represent most of the market. And doing _only_ sales into big companies would be really tough - slower sales, a better product required, and a longer cycle time so reduced learning.
```

Comment:
The way the last sentence is structured is weird - your writing is generally very free flowing (writing as one would speak) but this last sentence is a bit too much